roles-in-tech/security.md

```diff
@@ -1,13 +1,14 @@
 # Security
 
-
 ![security locks](\roles-in-tech\security-locks.jpg)
 
 ### Projected Time
 90-180 minutes
 
 ### Prerequisites
-How the Internet Works Topic Outline
+- To become security engineer, one should pursue bachelors degree in computer or IT field or have experience in security field.
```

Comment:
If people reading this are reading because they are trying to take this free bootcamp, is there another way they could become a security engineer?

Reply:
While preparing for this role I came across this in many sites so I added it. I have changed it in the file, t is not necessary but it is recommended to have at least a bachelors degree in computer science or IT field. And also some knowledge of networking and programming is beneficial for a security engineer, therefore they may require a bachelors degree. Here are some links that I referred to:
[BECOME A SECURITY ENGINEER](https://www.cyberdegrees.org/jobs/security-engineer/)
[HOW DO I BECOME A CYBERSECURITY ENGINEER?](https://www.onlineengineeringprograms.com/faq/how-do-i-become-a-cybersecurity-engineer)

If there are any required changes, I am happy to solve it.